lbr.py

```diff
@@ -403,6 +409,38 @@ def header_only_str(l):
     if loop_ipc:
       read_sample.tick *= 10
       read_sample.stop = None
+  def edge_stats():
```

Comment:
Thanks for adding this function.
Please hoist it outside read_sample() and pass what's needed, like lines, to it.